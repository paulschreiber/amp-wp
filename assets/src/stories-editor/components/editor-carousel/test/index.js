/**
 * External dependencies
 */
import { shallow } from 'enzyme';

/**
 * Internal dependencies
 */
import EditorCarousel from '../';

describe( 'EditorCarousel', () => {
	it( 'should render block shortcuts', () => {
		const wrapper = shallow(
			<EditorCarousel />
		);

		expect( wrapper ).toMatchSnapshot();
	} );
} );
