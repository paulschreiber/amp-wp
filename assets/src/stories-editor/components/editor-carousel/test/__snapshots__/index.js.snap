// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditorCarousel should render block shortcuts 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
